### ISSUE TITLE
can we make other distros

### ISSUE THREAD

**Issue by Pierrestro** [linux-3ds/linux#90]
this the only distribution we can run I think that we can try other distros like debian, fedora, and other

**Comment by iAmInActions** [comment]
We have http://muellers-software.org/share/3ds-debian.tar.xz which is a raspbian port for the 3ds but its nowhere near finished atm. I still need to work on the kernel module for networking and uart. Ill update this thread once the new version is done.

**Comment by Pierrestro** [comment]
thank you my brother was wondering if that is possible but do it at least boot to the desktop

**Comment by ParzivalWolfram** [comment]
> We have http://muellers-software.org/share/3ds-debian.tar.xz which is a raspbian port for the 3ds but its nowhere near finished atm. I still need to work on the kernel module for networking and uart. Ill update this thread once the new version is done.

If you'd be willing to open-source your kernel drivers, it'd be much appreciated. I think i'm the only one still working on this project, and it's slow going when the kernel is as much of a mess as it is (and I don't know a ton of C.) If I had working wireless and more reliable SD drivers, I'd already have most popular distros built and ready to go.

**Comment by Pierrestro** [comment]
can i have a video of booting and install process by the networking is IMPOSSIBLE the 3ds network card start it self when the 3ds menu loads with is not the case when we boot 3rd party os

**Comment by ParzivalWolfram** [comment]
> can i have a video of booting and install process by the networking is IMPOSSIBLE the 3ds network card start it self when the 3ds menu loads with is not the case when we boot 3rd party os

Yes. We solve that by writing a custom driver to start and use the wireless module ourselves. We have to figure out how the card works, so we can start it outside the official OS. That's been the process for every other bit of hardware on the system so far.

**Comment by Pierrestro** [comment]
can i have video of booting process

**Comment by Wolfvak** [comment]
> can i have video of booting process

@Pierrestro please open another issue for this if needed, I'm closing this one because it's been answered -- basically any ARMv6k compatible distro can technically run, ignoring driver/memory/performance limitations.
